### PR TITLE
Add local development instructions

### PR DESCRIPTION
**Pull request by anthonydillon**
## QA
Check the text is ok.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,33 @@ This is the Vanilla Framework theme for brochure based websites.
 npm i
 gulp build
 ```
+## Vanilla brochure theme local development
+
+To develop on Vanilla itself, simply pull down the project and make changes.
+
+To test it locally using our recommended tooling, you need to first install dependencies:
```
Comment (barrymcgee) [inline README.md]: Full stop here, move colon to statement below

File: README.md [hunk 2]
```diff
@@ -8,3 +8,33 @@ This is the Vanilla Framework theme for brochure based websites.
...
+
+You need to have bundler installed
```
Comment (barrymcgee) [inline README.md]: Add colon here

File: README.md [hunk 3]
```diff
@@ -8,3 +8,33 @@ This is the Vanilla Framework theme for brochure based websites.
...
+
+``` bash
+gem install bundler
+```
+
+You need to have node and npm. To install follow the [installation instructions](https://docs.npmjs.com/getting-started/installing-node) on this page.
```
Comment (barrymcgee) [inline README.md]: Using 'this page' infers the instructions are within the README itself - reword to:

'To install, follow the [NPM installation instructions](https://docs.npmjs.com/getting-started/installing-node).'

File: README.md [hunk 4]
```diff
@@ -8,3 +8,33 @@ This is the Vanilla Framework theme for brochure based websites.
 npm i
 gulp build
 ```
+## Vanilla brochure theme local development
+
+To develop on Vanilla itself, simply pull down the project and make changes.
```
Comment (barrymcgee) [inline README.md]: Do you mean to say 'Vanilla brochure theme' here?

**Comment by anthonydillon** [comment]
@barrymcgee done, thanks.

**Pull request merged**